Cliqz/HomePanel/VPN/VPNEndPointManager.swift

```diff
@@ -71,42 +88,46 @@ class VPNEndPointManager {
         "ro" : NSLocalizedString("Romania", tableName: "Lumen", comment: "VPN country name for Romania"),
         "rs" : NSLocalizedString("Serbia", tableName: "Lumen", comment: "VPN country name for Serbia"),
         "ua" : NSLocalizedString("Ukraine", tableName: "Lumen", comment: "VPN country name for Ukraine")
-        ]
-    
+    ]
+
+
     private var countries = [VPNCountry]()
     
-    //MARK:- Singlton & Init
+    // MARK:- Singleton & Init
     static let shared = VPNEndPointManager()
     
     init() {
-        generateStaticCountries()
         getVPNCredentialsFromServer()
     }
     
-    private func generateStaticCountries() {
-        for id in ["us", "de", "ba", "bg", "fr", "gr", "in", "it", "ca", "hr", "nl", "at", "pl", "pt", "ro", "rs", "es", "tr", "ua", "hu", "uk"] {
-            if let name = CountriesLookup[id] {
-                self.countries.append(VPNCountry(id: id, name: name, endpoint: "", remoteID: ""))
-            }
-        }
-    }
-    
     private func getVPNCredentialsFromServer() {
-        VPNCredentialsService.getVPNCredentials { [weak self] (credentials) in
+        VPNCredentialsService.getVPNCredentials { [weak self] (credentials, error) in
+            guard error == nil, let credentials = credentials else {
+                NotificationCenter.default.post(
+                    name: VPNEndPointManager.countriesUpdateErrorNotification,
+                    object: nil,
+                    userInfo: ["error": error!]
```

Comment:
this error! is a bit dangerous, because the guard will fail also if error == nil && credentials == nil and in this case hard wrap will crash